plugins/afl-scoreboard/game_renderer.py

```
@@ -581,24 +591,283 @@ def _draw_recent_game_status(self, draw: ImageDraw.Draw, game: Dict) -> None:
+            return f"{weekday} {name} {day}" if weekday else f"{name} {day}"
+        return f"{name} {day}"
+
+    def _weekday_for(self, game: Optional[Dict]) -> str:
+        """Weekday abbreviation from the game's start time, or ''."""
+        if not game:
+            return ""
+        raw = game.get("start_time_utc") or game.get("start_time")
+        if not raw:
+            return ""
+        try:
+            start = raw if isinstance(raw, datetime) else datetime.fromisoformat(
```

Ruff (0.16.1)

[error] Undefined name `datetime`

(F821)